src/main/java/SoraUI.java

```diff
@@ -0,0 +1,223 @@
+            + "    \\   \\    |  |  |  | |      /      /  /_\\  \\   \n"
+            + ".----)   |   |  `--'  | |  |\\  \\----./  _____  \\  \n"
+            + "|_______/     \\______/  | _| `._____/__/     \\__\\";
+
+    /**
+     * Character used at the beginning of user input prompts
+     */
+    protected static final String PROMPT_SYMBOL = ">";
+
+    /**
+     * Different acknowledgement words for Sora's neutral replies
+     */
+    protected static final String[] ACKNOWLEDGEMENT_WORDS = {
+            "Okay",
+            "Alright",
+            "Gotcha",
+            "Got it"
+    };
+
+    /**
+     * Default parameters for printing formatting lines
+     */
+    protected static final int DEFAULT_LINE_LENGTH = 60;
+    protected static final String DEFAULT_LINE_CHAR = "-";
+
+    /**
+     * List of command keywords
+     */
+    protected static final String EXIT_COMMAND_KEYWORD = "bye";
+
+    protected static final String LIST_COMMAND_KEYWORD = "list";
+
+    protected static final String ADD_TODO_COMMAND_KEYWORD = "todo";
+    protected static final String ADD_EVENT_COMMAND_KEYWORD = "event";
+    protected static final String ADD_EVENT_OPTION_KEYWORD = "/at";
+    protected static final String ADD_DEADLINE_COMMAND_KEYWORD = "deadline";
+    protected static final String ADD_DEADLINE_OPTION_KEYWORD = "/by";
+    protected static final String MARK_TASK_AS_DONE_COMMAND_KEYWORD = "mark";
+    protected static final String UNMARK_TASK_AS_DONE_COMMAND_KEYWORD = "unmark";
+
+    /**
+     * List of response messages
+     */
+    protected static final String LIST_PRE_EXECUTION_RESPONSE =
+            "%s, here's a list of tasks that you have given to me:\n";
+    protected static final String EMPTY_LIST_RESPONSE =
+            "Hmm, my list is empty at the moment...";
+
+    protected static final String ADD_TASK_SUCCESS_RESPONSE =
+            "%s, I've added your new task to my list:\n";
+    protected static final String ADD_TASK_FAILURE_RESPONSE =
+            "Oops! Somehow I wasn't able to add your task to my list...\nSorry about that! (-ω-、)";
+
+    protected static final String MARK_TASK_DONE_SUCCESS_RESPONSE =
+            "%s, I've marked this task as done:\n";
+    protected static final String MARK_TASK_DONE_FAILURE_RESPONSE =
+            "Oops, I couldn't mark that task as done.\nSorry about that... (-ω-、)";
+
+    protected static final String UNMARK_TASK_DONE_SUCCESS_RESPONSE =
+            "%s, I've marked this task as not done:\n";
+    protected static final String UNMARK_TASK_DONE_FAILURE_RESPONSE =
+            "Oops, I couldn't mark that task as not done.\nSorry about that... (-ω-、)";
+
+    protected static final String COMMAND_NOT_UNDERSTOOD_RESPONSE =
+            "Oops! I can't understand what you've just typed...\nCould you try again?";
+
+    /**
+     * Prints a line on the console based on the default parameters defined in this Java class.
+     */
+    protected void printLine() {
```

Comment:
Maybe additional class could be applied for printing features instead of putting all of them into one class?